api/src/main/java/org/pmiops/workbench/cdr/dao/CBCriteriaDao.java

```diff
@@ -92,7 +196,7 @@ List<DbCriteria> findCriteriaByDomainAndTypeOrderByIdAsc(
               + "and code like upper(concat(:term,'%')) "
               + "and match(fullText, concat('+[', :domain, '_rank1]')) > 0 "
               + "order by c.count desc")
-  Page<DbCriteria> findCriteriaByDomainAndTypeAndCodeAndStandard(
+  Page<DbCriteria> findCriteriaByDomainAndCodeAndStandard(
```

Comment:
renamed many methods to reflect parameters passed in and the correct order of those parameters

Reply:
very nice...saw this but did not take a step to redo the order

Reply:
also did a version with SearchType Enum so we could say searcTerm.getSearchType==[NO_TERMS|MODIFIED_ONLY|ENDS_WITH_ONLY|MODIFIED_AND_ENDS_WITH] ... but then the select case statement used moire lines,,, :-)